py3status/modules/deadbeef.py

```diff
@@ -60,34 +60,36 @@ def get_status(self):
         try:
             # get all properties using ¥ as delimiter
             status = check_output(['deadbeef',
-                                   '--nowplaying',
-                                   self.delimiter.join(['%a',
-                                                        '%t',
-                                                        '%l',
-                                                        '%e',
-                                                        '%y',
-                                                        '%n'])])
+                                   '--nowplaying-tf',
```

Comment:
Looking at the changelog http://deadbeef.sourceforge.net/ `--nowplaying-tf` was broken before version 0.7.2.

Maybe we should check the version an only support `{isplaying}` for the new version.